third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -318,46 +317,110 @@ func resourceBigtableInstanceValidateDevelopment(diff *schema.ResourceDiff, meta
 	return nil
 }
 
+// resourceBigtableInstanceClusterReorderTypeList causes the cluster block to
```

Comment:
Would we be able to get rid of this entirely if we made cluster into a set? If so, let's get that on the 4.0.0 docket.

Reply:
Nope, because we need the `ForceNew` behaviour. https://github.com/hashicorp/terraform/issues/15420